randovania/games/dread/json_data/Artaria.txt

```diff
@@ -5399,10 +4939,10 @@ Extra - asset_id: collision_camera_081
 
 > Under left side blocks; Heals? False
   * Layers: default
+  > Door to Transport to Burenia
+      Screw Attack and After Artaria - Screw Attack Rotatable
   > Total Recharge
       Gravity Suit or Lay Cross Bomb or Perform WBJ
```

Comment:
The Cross Bomb should probably have Beginner IBJ to be consistent with all other uses. This can also now include Screw Attack + Before Rotatable as an option.